include/ofi_shm.h

```diff
@@ -151,7 +139,7 @@ union smr_cmd_data {
 	struct {
 		uint64_t	sar;
 	};
-	struct smr_ipc_info	ipc_info;
+	struct ipc_info	ipc_info;
```

Comment:
Please align field ipc_info with the other fields in this struct.

Reply:
Based on the view in the browser, all of those fields are aligned (to a specific tab stop).  Do you have tabs set correctly?

Reply:
I changed to chrome and find the issue you are saying. Fixed in the latest revision.